### ISSUE TITLE
Edit of default values

### ISSUE THREAD

**Issue by smarteshome2020** [ioBroker/ioBroker.devices#283]
**Is your feature request related to a problem? Please describe.**
I use the device Adpater first time and try to get a new order of my devices

**Describe the solution you'd like**
Some default valuse eg. Temperatur or Termostat has only a subset of needed Values. It is not possible to delete eg. SET , ACTUAL of subset an so i recieded some error in Log.

**Describe alternatives you've considered**
Rename of default Values or posibility to delete

**Additional context**
 
![image](https://github.com/ioBroker/ioBroker.devices/assets/97242825/22a2627d-3f99-422d-a25a-4f88a6f92cea)

**Comment by Garfonso** [comment]
So what errors did you get in log? In iobroker-log or in browser console?

Also I don't really get, what you want to do.

For a Thermostat-Device it is not supported to delete the SET state, because it is required... otherwise it can not be a Thermostat-Device, because it misses its core functionality. What kind of thermostat do you have that can not set a target temperature?

For a temperature sensor the same is true for the ACTUAL state, because that is where the temperature measurements are expected.

---- 

Wenn es für dich einfacher ist, auch gerne auf Deutsch. Hier nochmal mein Text auf Deutsch:
Welche Fehlermeldungen sind im log aufgetreten? Im ioBroker-log oder in der Browser-Konsole?

Außerdem verstehe ich nicht wirklich, was du tun möchtest.

Für ein Thermostat-Gerät ist es nicht möglich, den SET-Zustand zu löschen, da dieser erforderlich ist... Andernfalls kann es nicht als Thermostat-Gerät fungieren, da ihm seine Kernfunktionalität fehlt. Was für eine Art von Thermostat hast du, das keine Zieltemperatur einstellen kann?

Für einen Temperatursensor gilt dasselbe für den ACTUAL-Zustand, da dort die Temperaturmessungen erwartet werden.

**Comment by Garfonso** [comment]
Ja, für Thermostat brauchst du ein SET, also einen Zustand, bei dem die Zieltemperatur eingestellt werden soll. 

Hast du denn ein Thermostat? Oder nur einen Termperatursensor? Dann wäre Temperatur das richtige Gerät, da wäre dann die Zuordnung:
ACTUAL -> temperatur
SECOND -> humidity

available kannst du "offline" zuordnen, musst da aber die Lesefunktion `!val` nehmen (also herumdrehen).

Grundsätzlich ist der "Name" (es ist eigentlich die ID) egal. Die Rolle muss stimmen und was man damit macht, muss passen. Leider ist das nicht immer so offensichtlich, finde ich... :-/

**Comment by Garfonso** [comment]
Was hat das mit Influx zu tun? Influx kannst du auch für die Aliase aktivieren. Oder du guckst, ob die ursprünglichen States von rolle usw. passen und erkannt werden, wenn du raum & funktion vergibst, dann brauchst du keine Aliase.

**Comment by smarteshome2020** [comment]
Schon klar, bin mir aber nicht sicher ob ich dich richtig verstehe.Wenn der alias z.b. 0.alias.Wohnzimmer.Temperatursensor.set lautet ist doch der Name per default beim aktivieren von Influx derselbe. Ich wollte mir ja nur die Arbeit sparen und nochmals einen weiteren Namen zu pflegen. Hätte einfach den vom alias genommen was aber bei …SET wenig sinn macht. Aber OK, evtl habe ich ja einen Gedankenfehler :-)Von meinem iPhone gesendetAm 01.09.2023 um 17:35 schrieb Achim Königs ***@***.***>:﻿
Was hat das mit Influx zu tun? Influx kannst du auch für die Aliase aktivieren. Oder du guckst, ob die ursprünglichen States von rolle usw. passen und erkannt werden, wenn du raum & funktion vergibst, dann brauchst du keine Aliase.
